cypress/e2e/yasr/plugins/pivot-table/pivot-table-plugin.spec.cy.ts

```diff
@@ -0,0 +1,42 @@
+import DefaultViewPageSteps from '../../../../steps/default-view-page-steps';
+import {YasqeSteps} from '../../../../steps/yasqe-steps';
+import {YasrSteps} from '../../../../steps/yasr-steps';
+import {PivotTableStubs} from '../../../../stubs/pivot-table-stubs';
+import {PivotTableSteps} from '../../../../steps/pivot-table-steps';
+
+describe('Pivot table plugin', () => {
+  beforeEach(() => {
+    PivotTableStubs.stubD3JavaScript();
+    PivotTableStubs.stubJqueryJavaScript();
+    PivotTableStubs.stubJqueryUiJavaScript();
+    PivotTableStubs.stubJsapiJavaScript();
+    PivotTableStubs.stubLoaderJavaScript();
+    PivotTableStubs.stubPivotJavaScript();
+    DefaultViewPageSteps.visit();
+  });
+
+  it('should show table with query data', () => {
+    // When I am on page with ontotext-web-component on it,
+    // and execute a query
+    YasqeSteps.executeQuery();
+
+    // Then I expect the "Pivot table" to be visible.
+    YasrSteps.getPivotTablePluginTab().should('be.visible');
+
+    // When I open "Pivot Table" plugin
+    YasrSteps.openPivotPluginTab();
+
+    // Then I expect to see the pivot table
```

Comment:
Descriptions are a bit of confusing. First you expect the pivot table to be visible then you open the pivot table and in the end you again expect it to be visible.

Reply:
Fixed